src/lib/wallets/components/Albedo.svelte

```diff
@@ -0,0 +1,12 @@
+<script>
+    import Albedo from '../../../routes/connect/ui/wallets/Albedo';
+    import Wallet from '../WalletTemplate.svelte';
+    import { albedo } from '../../../assets/index';
+
+    const albedoClass = new Albedo();
```

Comment:
Albedo is the class, what you are instantiating is an object (of class Albedo), so `albedoClass` is not accurate.

```suggestion
    const albedo = new Albedo();
```